contracts/EncodeDAOCore.sol

```diff
@@ -17,12 +17,26 @@ contract EncodeDAOCore is ERC721URIStorage, AccessControl {
         IssueStatus status
     );
 
+    event IssueVotedOn(
+        address voter,
+        uint256 issueId,
+        bool decision
+    );
+
     /// Constants
     bytes32 public constant MODERATOR_ROLE = keccak256("MODERATOR_ROLE");
 
     /// State vars
     Issue[] private currentIssues;
     Counters.Counter private _issueIds;
+    mapping(uint256 => mapping(address => Vote)) private votesOnIssues;
```

Comment:
Hmm, this would require to loop through all the votes for every issue and find if its yes or no - may be easier to add a `votes` field for the issue struct and just increment it by one when voting for and then only permitting one vote to be cast per round of voting. Thoughts?

@martinloesethjensen I know you guys maybe talked about this or were going to, maybe you can provide some insight. 😄

Reply:
Ah shit. That's a problem I didn't think of